apps/cms/README.md

```diff
@@ -27,9 +27,11 @@ Start the server locally:
 nx serve cms
 ```
 
-Optionally, you can seed the database (while the server is running) to generate dummy content:
+Optionally, you can seed the database to generate dummy content.
 
 ```bash
+# While the server is runnning:
+
```

Comment:
That's a subtle change but a good one. Most people go straight to the code and that's a good reminder 👍